common/rpc/params.go

```diff
@@ -77,7 +77,7 @@ func NewParams(serviceName string, config *config.Config, dc *dynamicconfig.Coll
 		return Params{}, fmt.Errorf("inbound TLS config: %v", err)
 	}
 	outboundTLS := map[string]*tls.Config{}
-	for _, outboundServiceName := range service.List {
+	for _, outboundServiceName := range service.ShardedServicesList {
```

Comment:
this was to fix a startup issue where the services were not being configured correctly on start. It might be no longer necessary, I might remove it

Reply:
can you move it to a separate PR?

Reply:
yeah, good call